src/base/static/components/templates/right-sidebar.js

```diff
@@ -39,6 +39,15 @@ const RightSidebar = props => {
           places={props.places}
         />
       )}
+      {props.rightSidebarConfig.component === "ActivityStreamWithLegend" && (
+        <Fragment>
+          <MapLegendPanel config={props.rightSidebarConfig} themed={true} />
```

Comment:
nit: I think `isThemed` would be a bit more clear.

Or we may also want to consider naming this something like `isInsideRightSidebar`.